components/GlobalInit/GlobalInit.c

```diff
@@ -4,7 +4,8 @@ static const char *TAG = "Global init";
 extern QueueHandle_t BufQueue1;
 extern SemaphoreHandle_t GetResponseSemaphore;
 extern SemaphoreHandle_t FinishWifiConfig;
-
+SemaphoreHandle_t HaveSaveForWifiSemaphore = NULL;
```

Comment:
why the name of semaphores start with "Have"

Reply:
I change it "SaveWifiParamExistSemaphore"